code/src/Artist-name.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+
+const ArtistName = (props) => {
+  return (
+    props.artistsArray.map((artist, i, array) => {
+      return (
+        <a className="artist-link" key={artist.id} href={artist.external_urls.spotify}>
+          <p className="artist-name">
+            {array.length > 1 && array.length!==2 && i < array.length - 1  && i !== array.length-2  ? artist.name + ',  ' 
```

Comment:
Your code is working, but is not very readable - consider using normal if statements in such complex situations.
Ternary operators are great, but mostly when used in easy situation like if/else (so, when we have only 2 options)